src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1685,6 +1715,33 @@ private Symbol AsMemberOfResultType(Symbol symbol)
             return symbol;
         }
 
+        private static Symbol AsMemberOfTupleType(TupleTypeSymbol tupleType, Symbol symbol)
```

Comment:
>AsMemberOfTupleType [](start = 30, length = 19)

I don't see how this change is related to the goal of the commit. I think this helper deserves to be put into a separate PR and should be accompanied with thorough testing for all different kinds of members that we might run into. Please separate it.

Reply:
Removed.

---
In reply to: [169858421](https://github.com/dotnet/roslyn/pull/24935#discussion_r169858421) [](ancestors = 169858421)